core/src/main/java/com/topjohnwu/superuser/internal/JobImpl.java

```diff
@@ -96,15 +96,14 @@ public void submit(@Nullable Executor executor, @Nullable Shell.ResultCallback c
     @NonNull
     @Override
     public Shell.Job add(@NonNull InputStream in) {
-        if (in != null)
-            handlers.add(InputHandler.newInstance(in));
+        handlers.add(InputHandler.newInstance(in));
```

Comment:
This is Java, not Kotlin, and we cannot be sure that the user follows the annotation :D
This null check added due to a request from a long-time libsu user.
So please remove this specific commit, thank you!